src/java/org/apache/cassandra/service/paxos/PaxosUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.paxos;
+
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+import com.google.common.util.concurrent.Uninterruptibles;
+
+import org.apache.cassandra.config.CassandraRelevantProperties;
+import org.apache.cassandra.metrics.CASClientRequestMetrics;
+import org.apache.cassandra.utils.MBeanWrapper;
+
+public final class PaxosUtils implements PaxosUtilsMBean
+{
+    public static final PaxosUtils instance = new PaxosUtils();
+
+    private static final String MBEAN_NAME = "org.apache.cassandra.service:type=Paxos";
```

Comment:
Found `org.apache.cassandra.service:type=Paxos` more compact and meaningful when exposed via JMX. Can stick to the `package:class` convention like `org.apache.cassandra.service.paxos:PaxosUtils"` is you think it is better/safer.

Reply:
I don't care. There may be people who use the 'class name' convention to quickly figure out the mbean name seeing
`PaxosUtils implements PaxosUtilsMBean` and not having to look into impl details that use a different mbean name.

do as you think is best :shrug: